lms/templates/problem.html

```diff
@@ -42,16 +42,41 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
     </span>
     % endif
     </div>
-    % if check_button:
-    <button class="check ${ check_button } btn-brand" data-checking="${ check_button_checking }" data-value="${ check_button }">
-        <span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span>
+    <button type="button" class="submit ${ submit_button } btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
+        <span class="submit-label">${ submit_button }</span><span class="sr"> ${_("your answer")}</span>
```

Comment:
It is from legacy. What about the existing Reset/Save your answer? @cptvitamin I wonder if we need "your answer" at all for Submit/Save/Reset.